File: python/meteroid/models/o_auth_error_response.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel
from .o_auth_error_code import OAuthErrorCode


@dataclasses.dataclass
class OAuthErrorResponse(BaseModel):
    """OAuth 2.0 error response as per RFC 6749 Section 5.2"""

    error: OAuthErrorCode

    error_description: t.Optional[str] = None

    error_uri: t.Optional[str] = None

File: go/o_auth_error_response.go
// this file is @generated
package meteroid

// OAuth 2.0 error response as per RFC 6749 Section 5.2
type OAuthErrorResponse struct {
	Error OAuthErrorCode `json:"error"`

	ErrorDescription *string `json:"error_description,omitempty"`

	ErrorUri *string `json:"error_uri,omitempty"`
}
